src/peergos/shared/user/RandomSecretType.java

```diff
@@ -24,4 +24,9 @@ static RandomSecretType fromCbor(CborObject cbor) {
     public Type getType() {
         return Type.Random;
     }
+
+    @Override
+    public String getExtraSalt() {
```

Comment:
So `RandomSecretType` doesn't use an extra salt?

Reply:
Indeed, it is already random, so we are no improving anything by adding salt.

Reply:
👍